app/metrics.yaml

```diff
@@ -23,3 +23,152 @@ distribution:
+    notification_emails:
+      - dmu@mozilla.com
+    expires: "2020-05-01"
+  insert_types:
+    type: labeled_counter
+    lifetime: application
+    description: >
+      Counting how many domains and URL are visited in a session.
+    labels:
+      - type_link
+      - type_query
+      - voice_query
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2230
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568
+    notification_emails:
+      - dmu@mozilla.com
+    expires: "2020-05-01"
+durarion:
+  immersive_mode:
+    type: timing_distribution
+    lifetime: application
+    description: >
+      Counting how long users stay at the immersive mode.
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2230
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568
+    notification_emails:
+      - dmu@mozilla.com
+    expires: "2020-05-01"
+  window_lifetime:
+    type: timing_distribution
+    lifetime: application
+    description: >
+      Counting how long is a Window opened.
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2230
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568
+    notification_emails:
+      - dmu@mozilla.com
+    expires: "2020-05-01"
+controls:
+  tray_new_window:
+    type: counter
+    lifetime: application
+    description: >
+      Counting how many times users open new windows.
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2230
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568
+    notification_emails:
+      - dmu@mozilla.com
+    expires: "2020-05-01"
+  context_menu_long_pressed:
+    type: counter
+    lifetime: application
+    description: >
+      Counting how many times users do a long press on the context menu.
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2230
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568
+    notification_emails:
+      - dmu@mozilla.com
+    expires: "2020-05-01"
+  curve_mode_actived:
+    type: boolean
+    lifetime: application
+    description: >
+      Counting how many users using the curved mode.
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2230
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568
+    notification_emails:
+      - dmu@mozilla.com
+    expires: "2020-05-01"
+searches:
+  engine:
```

Comment:
nit: this is not really an engine, this is the number of searches. Maybe `counts` is a better name?